roles/nuage_node/handlers/main.yaml

```diff
@@ -1,4 +1,7 @@
 ---
+- name: reload daemon
```

Comment:
can be also done with along with service restart. do we need separate task for this?

`systemd: name={{ openshift.common.service_type }}-node daemon-reload=yes state=restarted`